Source/SharpDX.Direct3D12/Mapping.xml

```diff
@@ -149,9 +156,19 @@
     
     <map field="D3D12_CLEAR_VALUE::Color" type="SHARPDX_VECTOR4" array="" />
     
+    <!-- Inner argument description structs have no name, so we specify it manually -->
+    <map struct="D3D12_INDIRECT_ARGUMENT_DESC_INNER_0_INNER_0" name="VertexBufferArgument"/>
+    <map struct="D3D12_INDIRECT_ARGUMENT_DESC_INNER_0_INNER_1" name="ConstantArgument"/>
+    <map struct="D3D12_INDIRECT_ARGUMENT_DESC_INNER_0_INNER_2" name="ConstantBufferArgument"/>
+    <map struct="D3D12_INDIRECT_ARGUMENT_DESC_INNER_0_INNER_3" name="ShaderResourceViewArgument"/>
+    <map struct="D3D12_INDIRECT_ARGUMENT_DESC_INNER_0_INNER_4" name="UnorderedAccessViewArgument"/>
+    <move struct="D3D12_INDIRECT_ARGUMENT_DESC_INNER_(.*)" to="D3D12_INDIRECT_ARGUMENT_DESC"/>
+    
+    
     <!--Remove all fields for D3D12_RESOURCE_BARRIER_DESC as the union with pointers cannot be mapped safely in x86/x64 with C# explicit layout-->
     <remove field="D3D12_RESOURCE_BARRIER::.*"/>
     <remove field="D3D12_ROOT_PARAMETER::.*"/>
+    <remove field="D3D12_TEXTURE_COPY_LOCATION::.*" />
```

Comment:
In this case, we don't want to remove the TextureCopyLocation but instead rename the field to "ResourcePointer", leave it internal and provide a setter only for the pResource as it is done for example for [ComputePipelineStateDescription](https://github.com/sharpdx/SharpDX/blob/master/Source/SharpDX.Direct3D12/ComputePipelineStateDescription.cs#L35)